src/cloud-api-adaptor/podvm-mkosi/Makefile

```diff
@@ -60,7 +68,8 @@ endif
 		--build-arg VERIFY_PROVENANCE=$(VERIFY_PROVENANCE) \
 		$(if $(AUTHFILE),--build-arg AUTHFILE=$(AUTHFILE),) \
 		$(if $(DEFAULT_AGENT_POLICY_FILE),--build-arg DEFAULT_AGENT_POLICY_FILE=$(DEFAULT_AGENT_POLICY_FILE),) \
-		-o type=local,dest="./resources/binaries-tree" \
+		$(if $(PUSH),,-o type=local,dest="./resources/binaries-tree") \
```

Comment:
@stevenhorsman , I didn't get this statement.

Also, does it work as intended?
```
wmoschet@wmoschet-thinkpadp16vgen1:/tmp$ cat Makefile 
PUSH ?= false
all:
	@echo $(if $(PUSH),,-o type=local,dest="./resources/binaries-tree")
wmoschet@wmoschet-thinkpadp16vgen1:/tmp$ make

wmoschet@wmoschet-thinkpadp16vgen1:/tmp$ make PUSH=false

```

Reply:
So the original code didn't push the binaries image to a container registry, but just stored it locally, but for performance reasons I wanted to cache it, so it could be re-used to subsequent builds if the binaries hadn't changed. That's why I added `PUSH` which sets the `--push` flag, like we have used elsewhere. I then figured that if we were pushing then we probably didn't need to also output to a local directory, so I tried to only enable this option if `PUSH` was false. I've probably messed up the syntax though, so I'll try and fix it.

Does that help & make sense, or do you think this is unecessary/incorrect in principle as well as execution?

Reply:
Ok, I've fixed the syntax error now:
```
root@sh-libvirt-x86:~# cat Makefile
PUSH ?= false

all:
	echo $(if $(filter $(PUSH),true),,-o type=local,dest="./resources/binaries-tree")
root@sh-libvirt-x86:~# make
echo -o type=local,dest="./resources/binaries-tree"
-o type=local,dest=./resources/binaries-tree
root@sh-libvirt-x86:~# PUSH=true make
echo

```